pkg/repo/repo.go

```diff
@@ -124,6 +123,9 @@ func (r *ChartRepository) Index() error {
 		r.IndexFile = &IndexFile{Entries: make(map[string]*ChartRef)}
 	}
 
+	var existCharts map[string]bool
+	existCharts = make(map[string]bool)
```

Comment:
The Go idiom for these two lines is probably better expressed as `exists := map[string]bool{}`